From 0578d9b2c250369b0014b12dc903a1d4c52bbabf Mon Sep 17 00:00:00 2001
From: "kirk.wang" <kirk.wang@amlogic.com>
Date: Fri, 1 Dec 2023 14:07:02 +0800
Subject: [PATCH] qt: aml-qt-rxpreview compilation error [1/1]

PD#SWPL-124930

Problem:
aml-qt-rxpreview demo

Solution:
add demo

Verify:
t3_ar301

Change-Id: Ie274ce00b4c0afa0c2417406835e68db524d32e1
Signed-off-by: kirk.wang <kirk.wang@amlogic.com>
---
 aml-qt-rxpreview/mainwindow.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/aml-qt-rxpreview/mainwindow.h b/aml-qt-rxpreview/mainwindow.h
index eed1655..0dfaf23 100644
--- a/aml-qt-rxpreview/mainwindow.h
+++ b/aml-qt-rxpreview/mainwindow.h
@@ -17,7 +17,6 @@
 #include <iostream>
 #include <gst/gst.h>
 #include <gst/video/videooverlay.h>
-#include "ImageDisplayThread.h"
 #include "Whiteboard.h"
 #include "Page1.h"
 #include "Page2.h"
-- 
2.25.1

